datasets/tedlium2.py

```diff
@@ -0,0 +1,222 @@
+                    f.write(" ".join(seg) + "\n")
+
+            f.close()
+            shutil.move(
+                "%s.stm" % corpus_key, self.out_stm_files[corpus_key].get_path()
+            )
+
+    def make_corpus(self):
+        """
+        create bliss corpus from stm file (always include speakers)
+        """
+        for corpus_key in ["train", "dev", "test"]:
+            audio_dir = os.path.join(self.corpus_folders[corpus_key].get_path(), "sph")
+            stm_file = self.out_stm_files[corpus_key].get_path()
+            data = self.load_stm_data(stm_file)
+
+            c = corpus.Corpus()
+            c.name = "TED-LIUM-realease2"
+
+            speakers = []
+            last_rec_name = ""
+            recording = None
+            for seg in data:
+                rec_name, channel, spk_name, start, end, gender, text = seg
+
+                if not spk_name in speakers:
+                    speakers.append(spk_name)
+                    speaker = corpus.Speaker()
+                    speaker.name = spk_name
+                    if "female" in gender:
+                        speaker.attribs["gender"] = "female"
+                    elif "male" in gender:
+                        speaker.attribs["gender"] = "male"
+                    c.add_speaker(speaker)
+
+                if rec_name != last_rec_name:
+                    if recording:
+                        c.recordings.append(recording)
+                    recording = corpus.Recording()
+                    recording.name = rec_name
+                    recording.audio = os.path.join(audio_dir, "%s.sph" % rec_name)
+                    last_rec_name = rec_name
+                    seg_id = 1
+
+                segment = corpus.Segment()
+                segment.name = str(seg_id)
+                segment.start = float(start)
+                segment.end = float(end)
+                segment.speaker_name = spk_name
+                segment.orth = text
+
+                recording.segments.append(segment)
+                seg_id += 1
+
+            if recording:
+                c.recordings.append(recording)
+            c.dump(self.out_corpus_files[corpus_key].get_path())
+
+    def load_stm_data(self, stm_file):
+        """
+        :param str stm_file
+        """
+        data = []
+        with uopen(stm_file, "r") as f:
+            for line in f:
+                if not line.strip() or line.strip().startswith(";;"):
+                    continue
+                if "ignore_time_segment_in_scoring" in line:
+                    continue
+                line = line.replace("<F0_M>", "<o,f0,male>")
+                line = line.replace("<F0_F>", "<o,f0,female>")
+
+                tokens = line.split()
+                assert len(tokens) >= 7, line
+                recName, channel, spkName, start, end, gender = tokens[:6]
+                text = " ".join(tokens[6:])
+                data.append([recName, channel, spkName, start, end, gender, text])
```

Comment:
```suggestion
                rec_name, channel, spk_name, start, end, gender = tokens[:6]
                text = " ".join(tokens[6:])
                data.append([rec_name, channel, spk_name, start, end, gender, text])
```
again prefer snake_case

Reply:
yep (again forgotten-changed copy from older pipeline)